packages/popover/src/Popover.js

```
@@ -62,13 +79,22 @@ class Popover extends React.Component {
   }
 
   static propTypes = {
-    trigger: PropTypes.node.isRequired,
+    trigger: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
     content: PropTypes.func.isRequired,
     align: PropTypes.oneOf(values(alignments)),
     zIndex: PropTypes.number,
     padding: PropTypes.string,
     overflow: PropTypes.string,
-    onOutsideClick: PropTypes.func
+    triggerWidth: PropTypes.string,
+    onOutsideClick: PropTypes.func,
+    onOpen: PropTypes.func,
+    onClose: PropTypes.func,
+    hideArrow: PropTypes.bool
```

What about `hideTooltipArrow`?

---

Or even `showTooltipArrow` since the default is to hide the arrow...